ajax/libs/scion/package.json

```diff
@@ -1,17 +1,91 @@
 {
-  "name": "scion",
-  "filename": "scion-min.js",
-  "version": "0.0.3",
+  "name": "scxml",
+  "version": "3.0.0",
   "description": "An implementation of SCXML in JavaScript.",
   "keywords": [
     "scxml",
     "statecharts",
     "w3c",
     "javascript"
   ],
+  "maintainers": [
```

Comment:
@jbeard4 We don't need the field like `maintainers`, `bugs`, `implements`, `devDependencies`, `engines`, `main`, `browser`. Please remove them, thank you. I think you can use `"author"` to replace with `"maintainers"`.